cla_backend/libs/eligibility_calculator/calculator.py

```diff
@@ -410,7 +415,7 @@ def _do_cfe_civil_check(self):
 
     @staticmethod
     def _is_applicant_detail_section_complete(case_data):
-        return hasattr(case_data.facts, "dependants_young") and hasattr(case_data.facts, "has_partner")
+        return case_data.facts.is_you_under_18 or (hasattr(case_data.facts, "dependants_young") and hasattr(case_data.facts, "has_partner"))
```

Comment:
@ripan Why is this not `self._under_18_passported(case_data)`? If they are under 18 passported, then I'd agree that's enough info to be calling CFE with. But if the applicant is under 18, but not under_18_passported, then we can't skip the other rules, I'd have thought

Reply:
Followed up in https://github.com/ministryofjustice/cla_backend/pull/1098